### PR TITLE
fix(styles): checkbox and radio border radius

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue
Part of #3327

## Description
* Changed border radius of the outline for the checkbox and radio button
* Refactored imports
* Refactored couple of mixins to use css variables

### Before:
image
image

### After:
image
image

### CODE REVIEW

File: src/styles/radio.scss [hunk 1]
```diff
@@ -60,18 +54,18 @@ $block: #{$fd-namespace}-radio;
...
+    padding: var(--fdRadio_Outer_Circle_Padding);
 
     &::before {
       content: "";
-      height: $fd-radio-inner-circle-diameter;
-      width: $fd-radio-inner-circle-diameter;
-      min-width: $fd-radio-inner-circle-diameter;
+      height: var(--fdRadio_Inner_Circle_Diameter);
+      width: var(--fdRadio_Inner_Circle_Diameter);
+      min-width: var(--fdRadio_Inner_Circle_Diameter);
```
Comment (InnaAtanasova) [inline src/styles/radio.scss:63]: this block of code is repeated 4 times. Maybe it's a good idea to create a mixin and use it in checkbox and radio button code.

File: src/styles/checkbox.scss [hunk 2]
```diff
@@ -268,26 +266,26 @@ $fd-checkbox-tristate-compact-offset: 0.25rem;
 
   &--compact {
     > .#{$block}__text {
-      margin-left: $fd-checkbox-margin-compact;
+      margin-left: var(--fdCheckbox_Compact_Padding);
```
Comment (InnaAtanasova) [inline src/styles/checkbox.scss:269]: you can use `fd-set-margin-left` mixin and won't need the RTL

File: src/styles/checkbox.scss [hunk 3]
```diff
@@ -149,13 +147,13 @@ $fd-checkbox-tristate-compact-offset: 0.25rem;
   &__text {
     @include fd-reset();
     @include fd-ellipsis();
-    @include fd-set-margins-x($fd-checkbox-margin, 0);
+    @include fd-set-margins-x(var(--fdCheckbox_Padding), 0);
 
-    margin-left: $fd-checkbox-margin;
+    margin-left: var(--fdCheckbox_Padding);
```
Comment (InnaAtanasova) [inline src/styles/checkbox.scss:152]: there's a mixin that can be used and will avoid the RTL handle

**Comment by InnaAtanasova** [comment]
Screen Shot 2022-05-13 at 11 45 36 AM

In the deployed version I still don't see the rounded corners.

**Comment by InnaAtanasova** [comment]
Focus is now fixed but Radio Buttons are still broken in Horizon

**Comment by N1XUS** [comment]
Sorry. Didn't commit all changes.

**Review by mikerodonnell89 — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**